src/EndOfYearBanner/index.stories.tsx

```diff
@@ -17,6 +17,7 @@ export default {
     },
     argTypes: {
         ...coreArgTypes,
+        ...ophanComponentIdArgType,
```

Comment:
Nice! This reminded me of a thing I was thinking about earlier but forgot to mention. If this is required now we should make sure the `canRender` for this banner reflects that (i.e. `canRender` should return `false` if the `ophanComponentId` is missing.

Reply:
I just updated the `canRender` in 2c74bd7 to reflect this. Does that sound okay @michaelclapham?